content/en/Getting started/glossary.md

```diff
@@ -316,11 +330,19 @@ Contrast with [mitigate](#mitigate). This reflects how we use **remediate** at
 Cobalt, and differs slightly from the NIST definition of 
 [remediation](https://csrc.nist.gov/glossary/term/remediation).
 
+## Route
+
+In web applications, routing is a mechanism that determines which functionality is triggered when a user sends specific HTTP requests. A _route_ is a URL pattern associated with a handler that processes the request.
```

Comment:
When I first read this, I thought, "Is this part of our explanation of internal/external network tests?" I see your explanation, and I think I understand your intent.

I _assume_ we're talking about _software_ routing here (not network routing)

But your definition, which includes `URL` makes me think about hardware routing, again. I like the definition I see from https://freecontent.manning.com/routing-with-react-in-modern-front-end-web-applications/ (If you use it, cite it.)

(Idea only, i'm open to other options)

```suggestion
## Route (software)

In software, it’s a system for resource navigation. If you’re working in the browser, you might be familiar with routing as it relates to:

 - URLs
 - Resources, such as paths to images and scripts, and so on
 
On the server, matching incoming request paths (https://api.ifelse.io/users/1) to resources from a database. 
```